plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1975,10 +1975,9 @@ dialog.workspace.resources.import_structure_from_minecraft.message=<html><b>Sele
   <br>You need to create the structure in test Minecraft client environment<br>using structure block and the stucture will appear on this list.
 dialog.workspace.resources.import_structure_from_minecraft.title=Import structure
 action.workspace.resources.import_sound=Import OGG sound(s)...
-action.workspace.resources.import_java_animation=Import Java animation...
 action.workspace.resources.animations.help=Help with 3D model animations
 action.workspace.resources.import_java_model=Import Java 3D model...
-action.workspace.resources.import_java_model_animation=Import Java 3D model animation...
```

Comment:
*resources.import_java_model_animation is used in AnimationImportActions.JAVA. We use different strings for menu and for toolbar, as other resources. This PR breaks menu bar so this makes me wonder if the PR was tested?